api/nodes/serializers.py

```diff
@@ -11,7 +11,8 @@
 
 from api.base.utils import get_object_or_error, absolute_reverse, add_dev_only_items
 from api.base.serializers import LinksField, JSONAPIHyperlinkedIdentityField, DevOnly
-from api.base.serializers import JSONAPISerializer, WaterbutlerLink, NodeFileHyperLink, IDField, TypeField, JSONAPIListField
+from api.base.serializers import (JSONAPISerializer, WaterbutlerLink, NodeFileHyperLink, IDField, TypeField,
```

Comment:
Two lines in a row are both importing from `api.base.serializers`- why aren't these consolidated together?

Also, try to keep imports within the same line sorted in a predictable manner; it makes code easier to skim through (eg when trying to figure out whether something has already been imported).